tdrs-backend/tdpservice/settings/common.py

```diff
@@ -314,13 +301,6 @@ class Common(Configuration):
         "django.contrib.auth.backends.ModelBackend",
     )
 
-    # conditionally set which URI to go to
-    if "VCAP_APPLICATION" in os.environ:  # pragma: nocover
-        appjson = os.environ["VCAP_APPLICATION"]
-        appinfo = json.loads(appjson)
-        if len(appinfo["application_uris"]) > 0:
-            os.environ["BASE_URL"] = "https://" + appinfo["application_uris"][0] + "/v1"
```

Comment:
This isn't needed anywhere since we already dynamically determine the BASE_URL in the deployment scripts.